### PR TITLE
Dialyzer warn unknown

### PR DESCRIPTION
**Pull request by lucafavatella**

### CODE REVIEW

File: src/ktn_meta_SUITE.erl [hunk 1]
```diff
@@ -57,7 +57,7 @@ dialyzer(Config) ->
   Dirs = dirs(Config),
   Warnings =
     case test_server:lookup_config(dialyzer_warnings, Config) of
-      undefined -> [error_handling, race_conditions, unmatched_returns];
+      undefined -> [error_handling, race_conditions, unmatched_returns, unknown];
```
Comment (elvisci) [inline src/ktn_meta_SUITE.erl]: Line 60 is too long:       undefined -> [error_handling, race_conditions, unmatched_returns, unknown];.

**Comment by lucafavatella** [comment]
From https://github.com/inaka/sumo_rest/pull/73#issuecomment-284850381

File: src/ktn_meta_SUITE.erl (hunk 1 above)
Comment (lucafavatella) [reply src/ktn_meta_SUITE.erl]: ok, changing this...

**Comment by lucafavatella** [comment]
Changed line length and force-pushed.

@elbrujohalcon It is all yours now.

File: src/ktn_meta_SUITE.erl [hunk 2]
```diff
@@ -57,7 +57,10 @@ dialyzer(Config) ->
   Dirs = dirs(Config),
   Warnings =
     case test_server:lookup_config(dialyzer_warnings, Config) of
-      undefined -> [error_handling, race_conditions, unmatched_returns];
+      undefined -> [error_handling
```
Comment (elbrujohalcon) [inline src/ktn_meta_SUITE.erl]: If you're gonna use ROK-style, you have to add a space after [ here

File: src/ktn_meta_SUITE.erl [hunk 3]
```diff
@@ -57,7 +57,10 @@ dialyzer(Config) ->
...
+                   , race_conditions
+                   , unmatched_returns
+                   , unknown];
```
Comment (elbrujohalcon) [inline src/ktn_meta_SUITE.erl]: In ROK-style, the last ] goes below the commas in the next row.

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]

File: src/ktn_meta_SUITE.erl (hunk 2 above)
Comment (lucafavatella) [reply src/ktn_meta_SUITE.erl]: addressed in force-pushed commit

**Comment by lucafavatella** [comment]
@elbrujohalcon Shall be ok now.

**Review by elbrujohalcon — APPROVED** [review]

**Pull request merged**